Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Types/Merchant/DriverPoolConfig.hs

```diff
@@ -39,7 +39,10 @@ data DriverPoolConfigD u = DriverPoolConfig
     singleBatchProcessTime :: Seconds,
     tripDistance :: Meters,
     createdAt :: UTCTime,
-    updatedAt :: UTCTime
+    updatedAt :: UTCTime,
+    radiusShrinkValueForDriversOnRide :: Int,
+    driverToDestinationDistanceThreshold :: Meters,
+    driverToDestinationDuration :: Seconds
   }
```

Comment:
can you leave `createdAt` and `updatedAt` as last fields, for better readability?